code/lightna/lightna-redis/App/Storage/Redis.php

```diff
@@ -38,26 +49,153 @@ protected function connect(): self
         return $this;
     }
 
-    public function set(string $key, mixed $value): void
+    /**
+     * @throws \RedisException
+     */
+    public function set(string $key, mixed $value, array $tags = []): void
     {
-        $this->client->set($key, $value);
+        $this->multi();
+
+        try {
+            $this->setKeyValue($key, $value);
+            $this->createTags($key, $tags);
+
+            $this->exec();
+        } catch (\RedisException $e) {
```

Comment:
avoid \ at the beggining by importing namespace into file

Reply:
Fixed